docs/user_stories/capacity_calculation_rsc.md

```diff
@@ -5,37 +5,37 @@ layout: default
-- Short-term and very short-term adequacy forecast,
+- Coordinated security analysis
+- Outage planning coordination
+- Coordinated capacity allocation
+- Short-term and very short-term adequacy forecast
 - Individual and common grid modelling and data set delivery
 
 
-We are going to explain how to perform a coordinated capacity computation using PowSyBl and some specific developments. We want to ensure that flows across borders respect given maximum admissible values, while ensuring electricity security of supply. If some overloads are reported, a remedial actions optimization is called to find the cheaper solution to solve the constraints. Remedial actions can be either changing the tap of a PST (it modifies the impedance of the network and so the load flows) or generator redispatching.
+We are going to explain how to perform a coordinated capacity computation using Powsybl and some specific developments. We want to ensure that flows across borders respect given maximum admissible values, while ensuring electricity security of supply. If some overloads are reported, a remedial actions optimization is called to find the cheapest solution to solve the constraints. Remedial actions can be either changing the tap of a PST (it modifies the impedance of the network and so the load flows) or doing generator redispatching.
 
 # Workflow
 
-The first input data of this process is the network model, coming from UCTE or CIM-CGMES European exchange formats. We also need an object to define the security domain of the network, built from a CRAC file (for "Contingency list, Remedial Actions and additional Constraints"): it contains a contingencies list, the constraints to monitor and the available remedial actions to get rid of the constraints.   
+The first input data of this process is the network model, coming from UCTE or CIM-CGMES European exchange formats. We also need an object to define the security domain of the network, built from a CRAC file (**C**ontingency list, **R**emedial **A**ctions and additional **C**onstraints): it contains a contingencies list, the constraints to monitor and the available remedial actions to get rid of the constraints.   
 
 In this process, we also need two computation engines:
-- A loadflow calculation is launched before and after each contingency to identify overloads.
-- A sensitivity calculation: for each border, a sensitivity analysis determines the impact on the flow of a small variation of the PST angle, and that for all PSTs, and determines the impact on the flow of a small variation of the generation, and that for all generators.
+- A loadflow computation: it is launched before and after each application of a contingency to identify overloads.
+- A sensitivity computation: for each border, a sensitivity analysis determines, for all PSTs, the impact on the flow of a small variation of the PST angle, and, for all generators, the impact on the flow of a small variation of the power generation.
```

Comment:
I prefer previous sentences, but you can change "calculation" by "computation".

Reply:
Ok I reverted to your initial sentence.